spec/Currencies/CurrencyListSpec.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace spec\Money\Currencies;
+
+use Money\Currencies\CurrencyList;
+use Money\Currency;
+use PhpSpec\ObjectBehavior;
+
+final class CurrencyListSpec extends ObjectBehavior
```

Comment:
Looks like checking for the interface implementation is missing: https://github.com/moneyphp/money/blob/master/spec/Currencies/BitcoinCurrenciesSpec.php#L19